setup.py

```diff
@@ -100,5 +100,6 @@ def load_requirements(*requirements_paths):
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
```

Comment:
I think we need Django and DRF support classifiers.

Reply:
I have aded django and i think its enough